project_service.go

```diff
@@ -66,6 +66,21 @@ func (p *projectService) remove(ctx context.Context, name string) (int, error) {
 	return httpStatusCode, nil
 }
 
+func (p *projectService) purge(ctx context.Context, name string) (int, error) {
+	u := defaultPathPrefix + "projects/" + name + "/removed"
```

Comment:
Use `u := fmt.Sprintf("` instead

Reply:
@linxGnu Thanks, I will fix all similar cases in this file. :-)